spring-cloud-task-integration-tests/src/test/java/org/springframework/cloud/task/listener/BatchExecutionEventTests.java

```diff
@@ -240,7 +241,7 @@ public void receive(Object exceptionMessage) {
 	public static class ItemWriteEventsListenerBinding {
 
 		@StreamListener(Sink.INPUT)
-		public void receive(Object itemWrite) {
+		public void receive(String itemWrite) {
 			assertTrue("Message should start with '3 items'", itemWrite.toString().startsWith("3 items "));
```

Comment:
@mminella Does not seem like a side effect - basically the argument was always a String, so now the method is just treating it as such (StreamListener helps with the coercion in certain cases). Doesn't seem to make a difference when the argument is not actually being used.